### ISSUE TITLE
place:folder without name

### ISSUE THREAD

**Issue by rugk** [cadeyrn/bookmarks-organizer#3]
When looking for bookmarks without a name:
![grafik](https://cloud.githubusercontent.com/assets/11966684/26276447/0c9de6e4-3d78-11e7-9828-23a40364b47a.png)

Firefox 53.0.2

**Comment by cadeyrn** [comment]
I don't see this behaviour in any of my Firefox profiles. Can you reproduce in a new Firefox profile?

I don't understand why do you see these special folders as bookmarks, there are the default places for bookmarks and cannot be renamed, the should have a name. Sure, I can add a workaround to handle these special folders, but I would like to understand the problem. Do you have an idea what could be so special in your profile?

**Comment by rugk** [comment]
> Do you have an idea what could be so special in your profile?

I have no idea.

I tried it with another profile, which was previously synced with the same profile, but there I cannot reproduce it.

**Comment by cadeyrn** [comment]
Could you share a HTML and/or JSON export of your bookmarks with me? It could help to understand the problem. At least I hope so. It could also be something with the places.sqlite database but it contains more private data, so I hope a simple bookmarks export is enough to reproduce…

**Comment by rugk** [comment]
Maybe I could have a look if I know how to export the data…

**Comment by cadeyrn** [comment]
Bookmarks > Show all Bookmarks. The most right icon in the toolbar has a menu item "Backup" for a JSON Export and "Export Bookmarks to HTML" for a HTML export.

**Comment by rugk** [comment]
Actually that's a big file. So here is the beginning and end:
```
"guid":"root________","title":"","index":0,"dateAdded":1450876517605000,"lastModified":1495793179981000,"id":1,"type":"text/x-moz-place-container","root":"placesRoot","children":[{"guid":"menu________","title":"Lesezeichen-Menü","index":0,"dateAdded":1450876517605000,"lastModified":1495617497954000,"id":2,"type":"text/x-moz-place-container","root":"bookmarksMenuFolder","children"
[…]
{"guid":"q3uL-BQjgkjY","title":"Firefox: Über Ihren Browser","index":4,"dateAdded":[…]259000,"lastModified":[…]1259000,"id":1706,"type":"text/x-moz-place","uri":"about:firefox"}]}]}
```

**Comment by cadeyrn** [comment]
Hm… it looks okay. :( I have no idea what the reason is. I think I'll just skip bookmarks with place protocol. Thank you for your help.

**Comment by cadeyrn** [comment]
Fixed in version 1.1.0.
